<launch>

  <arg name="map_h" default="50.0" />
  <arg name="map_w" default="20.0" />
  <arg name="resolution" default="0.5" />
  <arg name="occ_map_grid_size" default="0.5" />
  <arg name="z_min_mapped" default="1.0" />
  <arg name="ref_frame" default="world" />
  <arg name="goal_position_x" default="1.0" />
  <arg name="goal_position_y" default="1.0" />
  <arg name="fly_height" default="2.0" />
  <arg name="security_distance" default="1.0" />
  <arg name="next_point_reached_dist" default="0.5" />
  <arg name="speed_controller" default="false" />
  <arg name="max_control_speed" default="1.0" />
  <arg name="x_safe_zone" default="0.0" />

  <arg name="filter_sensor" default="false" />

  <arg name="namespace"/>

  <group ns="$(arg namespace)">

  <node name="path_planner" pkg="path_planner" type="path_planner_node" output="screen" args="">
    <param name="map_h" type="double" value="$(arg map_h)"/>
    <param name="map_w" type="double" value="$(arg map_w)"/>
    <param name="occmap_resolution" type="double" value="$(arg resolution)"/>
    <param name="occ_grid_size" type="double" value="$(arg occ_map_grid_size)"/>
    <param name="z_min_th" type="double" value="$(arg z_min_mapped)"/>
    <param name="ref_frame" type="str" value="$(arg ref_frame)"/>
    <param name="goal_position_x" type="double" value="$(arg goal_position_x)"/>
    <param name="goal_position_y" type="double" value="$(arg goal_position_y)"/>
    <param name="fly_height" type="double" value="$(arg fly_height)"/>
    <param name="security_distance" type="double" value="$(arg security_distance)"/>
    <param name="next_point_reached_dist" type="double" value="$(arg next_point_reached_dist)"/>
    <param name="speed_controller" type="bool" value="$(arg speed_controller)"/>
    <param name="max_control_speed" type="double" value="$(arg max_control_speed)"/>
    <param name="x_safe_zone" type="double" value="$(arg x_safe_zone)"/>

  </node>

  </group>

</launch>
